### PR TITLE
Endpoint restructuring

### PR DESCRIPTION
**Pull request by weerd**
This PR does some quick updates to the endpoints for `/reportbacks` and `/signups` that are needed for some upcoming work with the `ReportbackUploader` and the signup components.

*NOTE:* I tried doing a bit of testing but it was difficult because I was having issues with reportbacks on Thor and with signups I wasn't sure how to test that via Postman without being auth'd 😬 

I'll be testing further as I use these endpoints via the app to wrap up the components I'm working on.

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -23,9 +23,21 @@
...
+ */
+
 // Reactions
 $router->post('reactions', 'ReactionController@store');
 $router->delete('reactions/{id}', 'ReactionController@delete');
 
-// Embeds
-$router->get('embed', 'EmbedController@index');
+// Reportbacks
+$router->resource('reportbacks', 'ReportbackController', ['except' => ['create', 'edit', 'destroy']]);
```
Comment (deadlybutter) [inline routes/web.php]: We haven't used these anywhere in the JS right? I know Dave made that Redux demo with it, but I'm assuming we don't have to maintain that atm?

Comment (deadlybutter) [inline routes/web.php]: oh my god im an idiot you did fix the JS stuff IGNORE ME

Comment (weerd) [inline routes/web.php]: Yeah, I need to update the `fetchReportbacks()` method. It's using the helper from Gateway, but that won't work until we setup the JWT token shiz in the future. Good call.

**Comment by weerd** [comment]
@deadlybutter just FYI, made the update to that `fetchReportbacks()` endpoint 😉

**Comment by deadlybutter** [comment]
👍

**Pull request merged**